man/systemd-detect-virt.xml

```diff
@@ -172,6 +172,11 @@
             <entry><varname>proot</varname></entry>
             <entry><ulink url="https://proot-me.github.io/">proot</ulink> userspace chroot/bind mount emulation</entry>
           </row>
+
+          <row>
+            <entry><varname>pouch</varname></entry>
+            <entry><ulink url="https://github.com/alibaba/pouch">Alibaba's Efficient Enterprise-class Rich Container Engine</ulink></entry>
```

Comment:
that sounds like an ad. "efficient", "enterprise-class", "rich". None of the other entries here use such word. Just say "Pouch Container Engine" or so

Reply:
Okay, thanks.